spec/quality_spec.rb

```diff
@@ -191,6 +191,58 @@ def check_for_specific_pronouns(filename)
     expect(error_messages.compact).to be_well_formed
   end
 
+  it "documents all used settings" do
+    exemptions = %w(
```

Comment:
@indirect do you think any of these should be documented right now? If so, I'm happy to do it